yarn.lock

```diff
@@ -14362,6 +14362,11 @@ yargs@^15.0.2, yargs@^15.1.0, yargs@^15.3.1:
     y18n "^4.0.0"
     yargs-parser "^18.1.2"
 
+yarn@^1.22.17:
```

Comment:
I don't think we should be changing `yarn.lock` in this PR since we're just adding a lint rule here